app/javascript/components/mygfw-login/component.jsx

```diff
@@ -23,14 +23,17 @@ const socialButtons = [
 
 class MyGFWLogin extends PureComponent {
   render() {
-    const { className, plain } = this.props;
+    const { className, plain, simple } = this.props;
 
     return (
       <div className={cx('c-my-gfw', { '-plain': plain }, className)}>
-        <p>
```

Comment:
This will be redundant so lets just remove it.

Reply:
But we have that text in other parts of the app ?